frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -378,8 +379,8 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
               //sets this pin as currently selected
               const convertedProperty = convertToProperty(
                 m.properties,
-                m.position.lat,
-                m.position.lng,
+                getIn(m.geometry.coordinates, '1') ?? m.position.lat,
```

Comment:
m.position is the position of the marker, however, during spiderfication the position of the marker on the map is offset from the actual lat/lng of the property. Using the actual coordinates from the geometry correctly highlights the property.

Reply:
using getIn to avoid throwing an exception if geometry is not populated for some reason.